image_writers_guide/sti.adoc

```diff
@@ -5,8 +5,142 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+The main advantage of using sti for building reproducible docker image ease of use
+for developer. To meet that criteria you, as a builder image author, should be aware
+of the two basic requirements for the best possible sti performance, these are:
+
+* link:[#Required image contents]
+* link:[#STI scripts]
+
+
+== Required image contents
+
+The build process consists of three fundamental elements which are combined into
+final docker image, the three are: sources, scripts and builder image. During the
+build process sti must place sources and scripts inside that builder image. To do
```

Comment:
I'm not sure if calling STI 'scripts' is the right thing. They don't have to be scripts, right? I can have assemble script that will be static Go binary or Java binary. My 'run' command can be a webserver as well. So how about 'STI commands' ?